numpy/core/src/multiarray/multiarraymodule.c

```diff
@@ -66,6 +67,22 @@ NPY_NO_EXPORT int NPY_NUMUSERTYPES = 0;
 
 #include "get_attr_string.h"
 
+#include "numpy/ufuncobject.h"
+#include "ufunc_object.h"
+/*
+ *****************************************************************************
```

Comment:
Might it be more logical to have this contained in a separate `umathmodule.h` file?

Reply:
I cleaned this up by removing many uneeded include directives